tests/test_tuning_local.py

```diff
@@ -8,13 +8,14 @@
 from sklearn import linear_model
 import pytest
 import arcus.azureml.environment.environment_factory as fac
+import arcus.azureml.experimenting.aml_trainer as tr
```

Comment:
Is this needed, I dont see an usage below.

Reply:
agreed - will remove in future PR